crates/query-engine/translation/src/translation/mutation/check_columns.rs

```diff
@@ -0,0 +1,85 @@
+//! Before inserting or updating, check the specified columns against the db schema.
+
+use crate::translation::error::Error;
+use query_engine_metadata::metadata::database;
+use query_engine_sql::sql;
+use std::collections::BTreeMap;
+
+/// Check that no columns are missing, and that columns cannot be inserted to
+/// are not inserted.
+pub fn check_columns(
```

Comment:
The commentary of this function still only talks about inserting, but it should also apply to updating, right?

Reply:
Yes, revised the commentary.